application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -154,8 +154,8 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
 
         String applicationId;
         String applicationName;
-        Integer maxUses = handleIntegerTypeOption(maxUsesOption);
-        Integer maxAge = handleIntegerTypeOption(maxAgeOption);
+        Integer maxUses = requiredIntOptionIfPresent(maxUsesOption);
+        Integer maxAge = requiredIntOptionIfPresent(maxAgeOption);
```

Comment:
The variable name should include the unit to prevent bugs `maxAgeDays`

Reply:
Now that you made it days, the code will have a bug when you interpret it without additional logic, since you hand the input 1:1 over to `.setMaxAge(maxAge)`, which expects seconds (no?).

Reply:
What's the best way of doing this? Should I convert the units from days to seconds? as you're right and discord expects seconds.